gulp/tasks/pl_scripts.js

```diff
@@ -12,6 +12,9 @@ gulp.task('pl_scripts', ['pl_scripts-lint'], function() {
             config.pl_src + '/ui.js',
             config.pl_src + '/size-slider.js',
             config.pl_src + '/color-contrast.js',
+            config.pl_src + '/select-replace.js',
```

Comment:
if we are going to use these to style the forms, should they be separated from the pattern library rendering js? probably worth a conversation to talk about js and pattern library ... might make it harder to bring over because of conflicts with existing LMS js...